src/type-logic.ts

```diff
@@ -571,7 +574,7 @@ export function typeFromExprType(type: BiblioType): Type {
     }
     case 'unused': {
       // this is really only a return type, but might as well handle it
```

Comment:
This comment no longer makes sense now that we handle return types.

```suggestion
```

Reply:
Done.